ui/app/selectors/custom-gas.js

```diff
@@ -254,7 +269,7 @@ export function getRenderableBasicEstimateData (state, gasLimit, useFastestButto
   }
 
   return useFastestButtons
-    ? [averageEstimateData, fastEstimatData, fastestEstimateData]
```

Comment:
While we're here, let's fix the `fastEstimatData` typo:
```suggestion
    ? [fastEstimateData, fastestEstimateData]
```

Reply:
There's also a similar typo in the same file: `slowEstimatData`

Reply:
typos have been fixed in latest commit